hoomd/md/EvaluatorPairMie.h

```diff
@@ -75,9 +77,11 @@ class EvaluatorPairMie
                 Scalar epsilon = v["epsilon"].cast<Scalar>();
                 Scalar sigma = v["sigma"].cast<Scalar>();
 
-                Scalar outFront = (m3/(m3-m4)) * pow(m3/m4, m4/(m3-m4));
-                m1 = outFront * epsilon * pow(sigma, m3);
-                m2 = outFront * epsilon * pow(sigma, m4);
+                Scalar outFront = (m3/(m3-m4)) * fast::pow(m3/m4, m4/(m3-m4));
+                m1 = outFront * epsilon * fast::pow(sigma, m3);
+                m2 = outFront * epsilon * fast::pow(sigma, m4);
+                mie_log2_n = fast::log2(m3) / Scalar(2.0);
+                mie_log2_m = fast::log2(m4) / Scalar(2.0);
```

Comment:
```suggestion
                mie_log2_n = fast::log2(m3) / Scalar(2.0);
                mie_log2_m = fast::log2(m4) / Scalar(2.0);
```
With the new fast::pow refactoring, `mie_log2_n/m` are no longer used in `evalForceAndEnergy` and are not needed.